Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CallbackMapProperties.java

```diff
@@ -91,6 +91,16 @@ public String getUsername() {
      * @see gov.hhs.fha.nhinc.callback.openSAML.CallbackProperties#getAuthenicationContextClass()
      */
     @Override
+    public DateTime getIssueInstant() {
```

Comment:
How is issue instant populated?  Feel like this will always be null.  Not sure if this value is needed for this issue anyway.